relationships/synthesis/INFRA-KUBERNETES_POD-to-EXT-ENVOY.stg.yml

```diff
@@ -0,0 +1,43 @@
+relationships:
+  - name: k8sPodContains
+    version: "1"
+    origins: 
+      - Kubernetes Integration
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Metric" ]
+      - attribute: entity.type
+        anyOf: [ "ENVOY" ]
+    relationship:
+      expires: P75M
+      relationshipType: CONTAINS
+      source:
+        buildGuid:
```

Comment:
you don't need to use buildGuid on both source and target, if the entity guid comes in the telemetry you can just extract the guid from there, like [this](https://github.com/newrelic/entity-definitions/blob/main/relationships/synthesis/APM-APPLICATION-to-DATABASE.yml#L14-L15)